content/python/concepts/built-in-functions/terms/any/any.md

```diff
@@ -18,50 +18,91 @@ The `any()` built-in function takes in an iterable object such as a list or tupl
 
 ## Syntax
 
-```py
+```pseudo
 any(iterable)
 ```
 
-## Example 1
+The `iterable` is any sequence or collection that can be traversed, such as a [dictionary](https://www.codecademy.com/resources/docs/python/dictionaries) or [list](https://www.codecademy.com/resources/docs/python/list).
 
-Use `any()` to check if there is a `True` statement in a list `[True, 0, False]`:
+The inside of the pseudocode can be broken down in the following way:
 
-```python
-print(any([True, 0, False]))
-# Output: True
+```pseudo
+def any(iterable):
+  for element in iterable:
+    if element:
+      return True
+  return False
 ```
 
-## Example 2
+Inside the loop, at the first instance of an `element` existing in the `iterable`, execution will stop and `True` will be returned. If the end of the loop is reached and the element is not found, `False` is returned instead.
 
-Use `any()` to check if there is a `True` statement in a dictionary:
+## Example
 
-```python
-my_dict = {1 : "On"}
+In the following example, the any() function is used to return a boolean after checking a list and a dictionary:
 
-print(any(my_dict))
-# Output: True
+```py
+print(any([True, 0, False]))
+print(any({0 : "Off"}))
 ```
 
-## The Function
-
-The `any()` built-in function is essentially:
+The following output will be printed to the shell:
 
-```python
-def any(iterable):
-  for element in iterable:
-    if element:
-      return True
-  return False
+```shell
+True
+False
 ```
 
 ## Codebyte Example
 
-```codebyte/py
-# List
-my_list = [0, 2, False]
-print(any(my_list))
-
-# Dictionary
-my_dict = {0 : "Off"}
-print(any(my_dict))
+In the example below, a team of Pokemon are created in preparation for a battle. They are selected based on various properties such as `"level"` and `"type(s)"`. The `any()` function is ultimately used to pick out the Pokemon that meet that criteria:
+
+```codebyte/python
```

Comment:
Can we add a couple more comments to this codebyte and change the existing ones slightly? I've added suggestions but change them around as you see fit.